packages/config/src/layer2s/linea.ts

```diff
@@ -209,9 +209,7 @@ export const linea: Layer2 = {
       url: 'https://api.lineascan.build/api',
       type: 'etherscan',
     },
-    // ~ Timestamp of block number 0 on Linea
-    // https://lineascan.build/block/0
-    minTimestampForTvl: UnixTime.fromDate(new Date('2023-07-06T14:00:00Z')),
+    minTimestampForTvl: UnixTime.fromDate(new Date('2023-07-19T14:00:00Z')),
```

Comment:
why this got changed?

Reply:
Linea was basically dead until this 19th  (screenshot from staging)
![image](https://github.com/l2beat/l2beat/assets/72789647/434038bf-baaf-4f89-86a2-304493796596)